test.js

```diff
@@ -503,6 +503,21 @@ test('alias - require previousId and userId', t => {
   })
 })
 
+test('retryCondition', t => {
+  const client = createClient()
+
+  t.false(client._retryCondition({}))
+
+  // ETIMEDOUT is retryable as per `is-retry-allowed` (used by axios-retry in `isNetworkError`).
+  t.true(client._retryCondition({ code: 'ETIMEDOUT' }))
+
+  // ECONNABORTED is not retryable as per `is-retry-allowed` (used by axios-retry in `isNetworkError`).
+  t.false(client._retryCondition({ code: 'ECONNABORTED' }))
+
+  t.true(client._retryCondition({ response: { status: 500 } }))
+  t.true(client._retryCondition({ response: { status: 429 } }))
```

Comment:
We should add one last test to verify the default `return false` behavior.

Reply:
Added one, though it's pretty contrived; we'd never really expect to see and error for  a 2xx response.